addons/sale_pdf_quote_builder/i18n/sale_pdf_quote_builder.pot

```diff
@@ -37,7 +37,7 @@ msgid ""
 "e.g. this option can be useful to share Product description files.\n"
 "Confirmed order: the document will be sent to and accessible by customers.\n"
 "e.g. this option can be useful to share User Manual or digital content bought on ecommerce. \n"
-"Inside quote: The document will be included in the pdf of the quotation between the header pages and the quote table. "
+"Inside quote: The document will be included in the pdf of the quotation and sales order between the header pages and the quote table. "
```

Comment:
```suggestion
"Inside quote: The document will be included in the pdf of the quotation \n"
"and sales order between the header pages and the quote table. "
```

Reply:
@Demesmaeker why the `\n` in the middle of the sentence here ?

Reply:
Honestly? Because I probably did go too fast, was thinking "wrap it before the 100th chars", and probably messed up >.<